src/queries/index.js

```diff
@@ -76,6 +80,7 @@ export * from './types';
 export const getQuery = (query, filterOptions = {}) => {
   const QUERIES = {
     [QUERY_TYPES.CATEGORIES_FILTER]: GET_CATEGORIES_FILTER,
+    [QUERY_TYPES.CATEGORIES_FOR_PERSONALIZED_PUSH]: GET_CATEGORIES_FOR_PERSONALIZED_PUSH,
```

Comment:
we do not need the additional code everywhere. let's use the existing category filter query. i will update it.